package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "@dsnp/activity-content",
+  "version": "1.0.0",
+  "description": "",
+  "main": "dist/index.js",
+  "types": "dist/types/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/LibertyDSNP/activity-content.git"
+  },
+  "license": "ISC",
+  "scripts": {
+    "build:cjs": "tsc -p tsconfig.json",
+    "ts-types": "tsc --emitDeclarationOnly --outDir ./dist/types",
+    "build": "mkdir -p ./dist/types/ && esbuild --format=cjs ./src/index.ts src/core/index.ts --bundle --outdir=./dist --outbase=src && npm run ts-types",
+    "clean": "rm -Rf dist ./*.d.ts ./*.js ./*.tgz",
+    "doc": "rm -Rf dist/docs && npm run doc:markdown",
```

Comment:
I got some warnings when I ran this:

```
Warning: __module, defined at src/core/errors.ts:5, is referenced by core.errors but not included in the documentation.
Warning: __module, defined at src/core/factories.ts:0, is referenced by core.factories but not included in the documentation.
Warning: __module, defined at src/core/validation.ts:0, is referenced by core.validation but not included in the documentation.
```

Reply:
https://typedoc.org/guides/doccomments/#%40module

Reply:
https://typedoc.org/guides/options/#entrypointstrategy

New typedoc version changed the way it checks the files within the entrypoint. I added in an option to go back to the original way so that the errors go away.